src/scripts/components/noticeImage.js

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@wordpress/element';
+
+import wpLogo from '../../images/WordPressLogo.svg';
+
+export default class NoticeImage extends Component {
```

Comment:
I have a few thoughts on this, but most can wait for a followup later.

Each of these components could do with a comment though.